.docker/source/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# moveit/moveit:kinetic-source
+# Downloads the moveit source code, install remaining debian dependencies, and builds workspace
+
+FROM osrf/ros:kinetic-desktop
+MAINTAINER Dave Coleman dave@dav.ee
+
+ENV CATKIN_WS=/root/ws_moveit
```

Comment:
the default user is root, there is no sudo by default in a Ubuntu docker container